### DISCUSSION TITLE
How to enable custom themes *only* for dark mode, and don't apply any theme in light mode?

### DISCUSSION THREAD

**Discussion by pyinto** [openstyles/stylus#1839] [Q&A] [answered]
I'm using Stylus themes on various sites, but the thing is, I use [those themes](https://github.com/catppuccin/userstyles) only in dark mode. For light mode, I would prefer to use the native page with no CSS customizations. Any ideas on how to achieve that? In the settings, I can only set a custom flavor for dark/light mode, but there is no option to disable it at all.
Appreciate your help!

**Comment by tophf** [comment, marked as answer]
1. Open the style in the editor -> Style settings panel -> [x] Dark
2. Open Stylus options -> set the dark mode

**Reply by pyinto** [reply]
Oh, great, I didn't see that! The only thing is that I will have to go and do that on all my themes :D Hope it won't reset after the update. Thank you for the swift answer anyway!
